src/renderer/components/wallet/Settings.tsx

```diff
@@ -148,10 +131,17 @@ const Settings: React.FC<Props> = (props: Props): JSX.Element => {
           <StyledCard>
             <Row>
               <Col span={24}>
-                <StyledPlaceholder>{intl.formatMessage({ id: 'setting.midgard' })}</StyledPlaceholder>
-                <StyledClientLabel>128.128.128.128:8080</StyledClientLabel>
+                <StyledPlaceholder>{network === Network.MAIN ? 'Binance chain' : 'Midgard API'}</StyledPlaceholder>
```

Comment:
The title is still `MIdgard` (`setting.midgard`), but not `Binance chain` in case of `mainnet`.

Reply:
hm. and what is a title in case of testnet ?

Reply:
It's still `Midgard`.